std_msgs/Header header

uint8 id # 254 for station

float64 target_latitude
float64 target_longitude
float64 target_altitude

geometry_msgs/Vector3 target_position_body_frame

float64 target_position_in_photo_x
float64 target_position_in_photo_y





